csharp/Android/Facebook.Yoga.Android/CustomBuildAction.targets

```diff
@@ -0,0 +1,16 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
+<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+	<PropertyGroup>
+		<CompileDependsOn>CopyInNativeLib;$(CompileDependsOn)</CompileDependsOn>
```

Comment:
<?xml version="1.0" encoding="UTF-8"?>
    <Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
    	<PropertyGroup>
            <CompileDependsOn>NativeLibrary;$(CompileDependsOn)</CompileDependsOn>
        </PropertyGroup>
        <Target Name="NativeLibrary" Outputs="$(ProjectDir)libyoga.a">
            <Exec WorkingDirectory="$(ProjectDir)" Command="$(ProjectDir)../../Mac/buck-build.sh //yoganet#android-armv7,shared" />
            <Exec WorkingDirectory="$(ProjectDir)" Command="$(ProjectDir)../../Mac/buck-build.sh //yoganet#android-x86,shared" />
            <Copy SourceFiles="$(ProjectDir)..\..\..\buck-out\gen\csharp\yoganet#android-armv7,shared\libyoga.so" DestinationFiles="$(ProjectDir)/lib/armeabi-v7a/libyoga.so" SkipUnchangedFiles="true" />
            <Copy SourceFiles="$(ProjectDir)..\..\..\buck-out\gen\csharp\yoganet#android-x86,shared\libyoga.so" DestinationFiles="$(ProjectDir)/lib/x86/libyoga.so" SkipUnchangedFiles="true" />
        </Target>
    </Project>

and I've realized we're using `/` and `\` for path in Mac and iOS as well...